doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -888,6 +888,16 @@ The specific use case of the Google Fast Pair application is indicated by the ch
 In the official `Fast Pair`_ documentation, the `Fast Pair Device Feature Requirements`_ category defines additional requirements for each supported use case, and specifies a list of mandatory, optional, and unsupported Fast Pair features.
 If your product is targeting one of the listed use cases, you must align your accessory firmware to meet these requirements.
 
+If your Google Fast Pair use case is supported by the |NCS|, you can select it using the :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE` Kconfig choice option.
```

Comment:
Maybe we should mention currently supported use cases?

Reply:
They are listed in subsequent sections (currently only the "Locator Tag"):

```In this case, you can find more specific description in the following subsections that describe Fast Pair use cases supported by the |NCS|.```

Reply:
Hmm, I could add a sentence:

```
You can select one of the following use case options:

* :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_UNKNOWN`: the Unknown use case (the default choice) - see the :ref:`ug_bt_fast_pair_use_case_unknown` subsection for more information.
* :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_LOCATOR_TAG`: the Locator Tag use case - see  the :ref:`ug_bt_fast_pair_use_case_locator_tag` subsection for more information.
```
What do you think about it?

Reply:
Looking at how doc renders, the list of use-cases is generated in side panel located at the right: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/external_comp/bt_fast_pair.html. I doubt if it's necessary to add the list explicitly also here, but I would not block because of it.

Reply:
You have to know that it is there - there is a lot of other items on that list, and also you have to click dropdown button. I wouldn't relay on that.

Reply:
applied this suggestion